### ISSUE TITLE
Modifier lock does not work

### ISSUE THREAD

**Issue by mpmuskie** [Oire/TheQube#72]
Hello,
When using The Qube, the modifier lock does not work. I have toggled the modifier lock on by pressing alt + win + k, and by executing the funcktion from the keyboard manager. In both instances, pressing of the up or down arrow keys does not read tweets from the current buffer, but rather act as they would normally. I have tried this from different windows, including internet, desktop, and windows explorer with the same results. One of the reasons I love The Qube is the modifier lock! I am using The Qube 0.9.1, although I did experience the same issue with The Qube 0.8, Jaws 17 and Windows 10 64-bit. 
Thanks,
MP

**Comment by Menelion** [comment]
@mpmuskie, good catch, thanks! I believe that's because I had to change the modifiers from Ctrl+win to Alt+Win for Windows 10 compatibility. I will investigate.
